clamd/session.c

```diff
@@ -591,8 +591,10 @@ int execute_or_dispatch_command(client_conn_t *conn, enum commands cmd, const ch
         }
         case COMMAND_INSTREAM: {
             int rc = cli_gentempfd(optget(conn->opts, "TemporaryDirectory")->strarg, &conn->filename, &conn->scanfd);
-            if (rc != CL_SUCCESS)
-                return rc;
+            if (rc != CL_SUCCESS) {
+                logg(LOGG_ERROR, "Failed to generate file in TemporaryDirectory, filename: %s \n", &conn->filename);
```

Comment:
There should already be a log message when `cli_gentempfd` fails, but I suppose an additional error is okay. 

https://github.com/Cisco-Talos/clamav/blob/7d26bc8f6a801bb35ff7f2b154f33817e5863ea6/libclamav/others_common.c#L1249-L1322

Reply:
o7, resolving comment then